packages/core/test/interpreter.test.ts

```diff
@@ -1843,3 +1843,49 @@ it(`should execute entry actions when starting the actor after reading its snaps
 
   expect(spy).toHaveBeenCalled();
 });
+
+it('ref', (done) => {
```

Comment:
those two added test cases could use meaningful titles 😉 we should also move them to the main `describe` block - the auto-sorting of the test summary is super confusing. Even though those are at the end of the file, they are on the top in the test summary:
```
 PASS  packages/core/test/interpreter.test.ts
  ✓ should execute entry actions when starting the actor after reading its snapshot first
  ✓ ref (1 ms)
  ✓ ref2
  interpreter
    ✓ can cancel a delayed event (1 ms)
    ✓ can cancel a delayed event using expression to resolve send id (202 ms)
    ✓ should throw an error if an event is sent to an uninitialized interpreter if { deferEvents: false } (5 ms)
    ✓ should not throw an error if an event is sent to an uninitialized interpreter if { deferEvents: true }
    ✓ should not throw an error if an event is sent to an uninitialized interpreter (default options) (1 ms)
    ✓ should defer events sent to an uninitialized service
    ✓ should throw an error if initial state sent to interpreter is invalid (1 ms)
    ✓ should not update when stopped (23 ms)
    ✓ should be able to log (log action) (1 ms)
    ✓ should be able to log event origin (log action) (2 ms)
    ✓ should receive correct _event (log action)
    ✓ shouldn't execute actions when reading a snapshot of not started actor
    initial state
      ✓ immediately notifies the listener with the initial state and event (3 ms)
      ✓ .initialState returns the initial state (1 ms)
      ✓ initially spawned actors should not be spawned when reading initial state (102 ms)
      ✓ executes actions from a restored state (2 ms)
      ✓ should not execute actions that are not part of the actual persisted state (1 ms)
[...]
```

Reply:
Sorry about that - these tests were useful to help me fix some bugs in `@xstate/react` without going through the React plumbing 😂 I can remove them if you want me to.

Reply:
It might be worth keeping the first one (with a proper title etc) - the second one is based on a somewhat hacky implementation within `@xstate/react` since it changes the `.status` "manually". That probably shouldn't be something that people should do. I hope to refactor this in the `@xstate/react` too. So I'd say that the second one can be removed.